drivers/ps2/CMakeLists.txt

```diff
@@ -3,3 +3,5 @@
 zephyr_library()
 
 zephyr_library_sources_ifdef(CONFIG_PS2_XEC		ps2_mchp_xec.c)
+zephyr_library_sources_ifdef(CONFIG_PS2_NPCX		ps2_npcx_channel.c)
```

Comment:
This PR is missing a change into CODEOWNERS file
/drivers/ps2/*npcx*                        @ChiHual

Reply:
Thanks for the comment.  
I have added Nuvoton developers to the CODEOWNERS file in the 2nd commit of this PR.